src/main/java/pl/javastart/task/MotQueue.java

```diff
@@ -0,0 +1,23 @@
+package pl.javastart.task;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class MotQueue {
+
+    private static final Queue<Vehicle> vehicles = new LinkedList<>();
+
+    public static void addVehicleToQueue(String type, String make, String model, int year, int mileage, String vin) {
+        vehicles.add(new Vehicle(type, make, model, year, mileage, vin));
+    }
```

Comment:
W tej klasie również zmień wszystko ze static na normalne pola i metody. Używanie static to bardziej programowanie proceduralne, a nie obiektowe, ogólnie niewarto wyrabiać sobie tego nawyku kosztem tego, że jedynie unikniemy zapisania jednego `new`